### PR TITLE
Remove EnableSignedContainerCache + EnableMeasureMemoryMonitor from container launcher

### PR DESCRIPTION
**Pull request by yawangwang**
Remove unused EnableSignedContainerCache + EnableMeasureMemoryMonitor from container launcher

### CODE REVIEW

File: launcher/agent/agent_test.go [hunk 1]
```diff
@@ -88,16 +87,9 @@ func TestAttest(t *testing.T) {
...
 	}{
 		{
-			name:                       "all experiment flags disabled",
-			launchSpec:                 spec.LaunchSpec{},
-			principalIDTokenFetcher:    placeholderPrincipalFetcher,
-			containerSignaturesFetcher: signaturediscovery.NewFakeClient(),
-		},
-		{
-			name: "enable signed container",
+			name: "all experiment flags disabled",
```
Comment (JoshuaKrstic) [inline launcher/agent/agent_test.go:90]: maybe we rename this now that theyre gone?
"Happy path" or something

**Review by JoshuaKrstic — APPROVED** [review]

File: launcher/agent/agent_test.go (hunk 1 above)
Comment (yawangwang) [reply launcher/agent/agent_test.go]: Done.

**Pull request merged**